src/filtrering/filtrering-status/filtrering-status.tsx

```diff
@@ -56,9 +56,28 @@ export function FiltreringStatus(props: FiltreringStatusProps) {
 
     const statusTall = useStatusTallSelector();
     const erVedtaksStotteFeatureTogglePa = useFeatureSelector()(VEDTAKSTOTTE);
+    const visMeldingOmBrukereMedAdressebeskyttelseEllerSkjerming =
+        useFeatureSelector()(VIS_MELDING_OM_BRUKERE_MED_ADRESSEBESKYTTELSE_ELLER_SKJERMING) &&
+        props.oversiktType === OversiktType.enhetensOversikt &&
+        statusTall.adressebeskyttelseEllerSkjermingTotalt &&
```

Comment:
Går det ann å kjøre de to siste som no slikt?
!!statusTall?.adressebeskyttelseEllerSkjermingTotalt > 0